main.go

```diff
@@ -7,30 +7,31 @@ import (
 	"github.com/hashicorp/go-hclog"
 	_ "github.com/lib/pq"
 	"github.com/turbot/go-kit/helpers"
-	"github.com/turbot/steampipe-plugin-sdk/logging"
 	"github.com/turbot/steampipe/cmd"
 	"github.com/turbot/steampipe/utils"
 )
 
 var Logger hclog.Logger
 
 func main() {
-	logging.LogTime("main start")
+	utils.LogTime("main start")
+	exitCode := 0
```

Comment:
there is a global exit code in root, so use that.

And this code in root: 

```
func Execute() {
	rootCmd.Execute()
	os.Exit(exitCode)
}
```
maybe that just needs to be a defer

Reply:
If we exit from `root.go` then the `defer` in `main.go` doesn't fire and the `DisplayProfileData` function is never executed.